docs/khadijah_docs.md

```diff
@@ -0,0 +1,71 @@
+
+##  تجربتي مع منصة الترجمة Crowdin
+
+تم إنشاء جمعية العلوم المفتوحة في المملكة العربية السعودية ** [OSCSA](https://osc-ksa.com/) تماشيًا مع الرؤية السعودية 2030 لدعم وتوفير بنية تحتية للعلوم المفتوحة في العالم العربي من خلال العمل مع الآخرين المجتمعات العلمية مثل Turing Way باستخدام [GitHub](www.github.com) ومنصة [Crowdin](www.crowdin.com)
+
+### البدء مع Crowdin
+
+<p> Crowdin هي أداة لإدارة التعريب تساعد الشركات والفرق والأفراد في الحفاظ على ذكائهم عند ترجمة منتجات مثل تطبيقات الأجهزة المحمولة وسطح المكتب والتطبيقات عبر الإنترنت والمواقع الإلكترونية والألعاب ومواد الدعم والمزيد.
```

Comment:
This paragraph is aligned left to the right - it should be right to the left, can you fix it?